kivy/uix/listview.py

```diff
@@ -899,12 +899,18 @@ def __init__(self, **kwargs):
                   item_strings=self.item_strings_changed,
                   adapter=self._trigger_populate)
 
+        self._trigger_bind_adapter = Clock.create_trigger(
+            lambda dt: self.adapter.bind_triggers_to_view(
```

Comment:
Every time what? This is in `__init__`, it will only be called once per widget. The resulting trigger will be a `ClockEvent` which will function the same regardless of whether the callback is a lambda or not.